tests/libres_tests/res/enkf/test_runpath_list_ert.py

```diff
@@ -28,7 +28,7 @@ def test_assert_export(self):
             runpath_list = ert.getRunpathList()
             self.assertFalse(os.path.isfile(runpath_list.getExportFile()))
 
-            ens_size = ert.getEnsembleSize()
+            ert.getEnsembleSize()
```

Comment:
Does the test pass without this?

Reply:
Yes, but choose to keep it in case it tests side effects.